framework/include/controls/RealFunctionControl.h

```diff
@@ -62,7 +62,7 @@ class RealFunctionControl : public Control
   Function & _function;
 
   /// Vector of parameters to change
-  std::vector<Real *> _parameters;
+  ControllableParameter<Real> _parameters;
```

Comment:
You misunderstood me... I mean that you shouldn't get a `ControllableParameter` object.  You should be able to get the value(s) of controllable parameters (note that that is not a class name).